taxonomies/categories.txt

```diff
@@ -62334,6 +62450,14 @@ it:Pasta senza glutine
 nl:Glutenvrije pastas
 pl:Makaron bezglutenowy
 
+<en:Gluten-free pasta
+<en:Dry pastas
+en:Gluten-free dry pasta, Dry gluten-free pasta, Dried gluten-free pasta
+fr:Pâtes sèches sans gluten
+agribalyse_proxy_food_code:en:9824
+agribalyse_proxy_food_name:en:Dried pasta, gluten-free, cooked, unsalted
```

Comment:
same here, it's not really a proxy, it's the actual match

Reply:
fixed